config/default.yml

```diff
@@ -990,6 +990,11 @@ Style/LambdaCall:
     - braces
 
 Style/MethodCallWithArgsParentheses:
+  Exclude:
```

Comment:
I'm not super excited about this, but am also aware of the enormous number of false positives from DSLs. I'm a bit on the fence about the cop being enabled by default or not. I really miss it in the cases where it catches inconsistencies though.